src/faldbt/project.py

```diff
@@ -620,6 +620,31 @@ def _lazy_setup_firestore(self):
                     "Could not find acceptable Default GCP Application credentials"
                 )
 
+    @telemetry.log_call("execute_sql")
+    def execute_sql(self, sql: str) -> pd.DataFrame:
+        """Execute a sql query."""
+        if lib.IS_DBT_V0:
+            raise NotImplementedError(
+                "execute_sql only supported in dbt version >= 1.0.0"
+            )
+
+        from dbt.lib import compile_sql
```

Comment:
our `faldbt/lib.py` file is actually inspired from `dbt.lib`. Do we want to import it from them or re-do it on our side?

Reply:
Great suggestion! But I think that it's better to let dbt handle the `compile_sql` function so we don't have to worry about it.